src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partnerSex = sex === 'f'
+    ? `${partnerName} is my husband`
+    : `${partnerName} is my wife`;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
```

Comment:
Move the content to the new line
```suggestion
      <h2 className="Person__name">
        {`My name is ${name}`}
      </h2>
```